deploy/stacks/lambda_api.py

```diff
@@ -161,6 +162,7 @@ def __init__(
             'envname': envname,
             'LOG_LEVEL': 'INFO',
             'email_sender_id': email_notification_sender_email_id,
+            'ALLOWED_ORIGINS': allowed_origins,
```

Comment:
don't think we will need to set `ALLOWED_ORIGINS` for awshandler lambda... this lambda only reads from a sqs queue and is not handling API requests

Reply:
removed this Noah.